docs/DeveloperGuide.md

```diff
@@ -287,6 +287,35 @@ The following activity diagram summarizes what happens when a user executes a ne
     * Pros: Time efficient as we can make use of the shorter prefixes because there's no need to differentiate between target appointment and new appointment details, Easier to correct mistakes in updating wrong appointment
     * Cons: Prone to errors as just a wrong number could cause the user to update the wrong appointment, Huge list could make it difficult to find a specific index
 
+
+### Mark/Unmark Appointment for a Patient
+
+#### Implementation
+
+The Mark Appointment feature will involve parsing user input and marking the appointment as completed/ not completed.
+
+The implementation will include the following key components:
+
+1. **Parsing User Input**: The application will parse user input to extract values to find the target appointment (NRIC, DATE, START_TIME, END_TIME).
```

Comment:
I think can remove mention of `END_TIME` in this so that it reflects the current implementation!